rholang/src/main/scala/coop/rchain/rholang/interpreter/Reduce.scala

```diff
@@ -1387,6 +1387,27 @@ class DebruijnInterpreter[M[_], F[_]](
       } yield result
   }
 
+  private[this] val hash: Method = new Method() {
```

Comment:
We already have a blake2b256 system process: https://github.com/rchain/rchain/blob/d0a3dd737828eca8ec9320b7d590d5421567879f/rholang/src/main/scala/coop/rchain/rholang/interpreter/SystemProcesses.scala#L240